src/Provider/Provider.Web/Configuration/Routing/RouteNames.cs

```diff
@@ -25,6 +25,7 @@ public static class RouteNames
         public const string Dashboard_YourCohorts = "Dashboard_YourCohorts";
         public const string Dashboard_AccountsAgreements = "Dashboard_AccountsAgreements";
         public const string Dashboard_Account_Home = "Dashboard_Account_Home";
+        public const string Dashboard_DismissAlert = "Dashboard_DismissAlert";
```

Comment:
Should this be suffixed with `_Post`

Reply:
👍 Will make all fixes on ER-1408_withdrawn_vacancies_provider branch